manifold.R

```diff
@@ -0,0 +1,30 @@
+#Manifold, i.e. a topological space that locally resembles
+#Euclidean space near each point.
+
+library(reticulate)
+gs <- import_from_path("geomstats", path = ".")
+
+Manifold <- setRefClass("Manifold",
+                        methods = list(
+
+                          init <- function(self, dimension){
+                            stopifnot(dimension %% 1 == 0, dimension > 0)
+                            self$dimension = dimension
+                          },
+
+                          belongs <- function(self, point){
+                            # Evaluate if a point belongs to the manifold.

```

Comment:
We need to understand how R handles 'abstract classes' or 'interfaces' with the framework of ReferenceClasses.

Reply:
But for now, if there's nothing in these functions, you can delete them.